lib/utils/CrossfilterHistogramPlot.ts

```diff
@@ -101,6 +104,62 @@ export function CrossfilterHistogramPlot(
 				.attr("dx", (_, i) => i === 0 ? "-0.25em" : "0.25em");
 		});
 
+	//~ Background rect for the next section (hover label)
+	svg.insert("rect")
+		.attr("transform", `translate(0,${height - marginBottom})`)
+		.attr("width", 20)
+		.attr("height", 20)
+		.style("fill", "white")
+		.attr("class", "hovered-bg");
+
+	// Value under cursor label
+	const hoveredTick = svg
+		.append("g")
+		.attr("transform", `translate(0,${height - marginBottom})`)
+		.call(
+			d3
+				.axisBottom(x)
+				.tickValues([0])
+				.tickFormat(tickFormatterForBins(type, bins))
+				.tickSize(2.5),
+		)
+		.call((g) => {
+			g.select(".domain").remove();
+			g.attr("class", "gray");
+			g.selectAll(".tick text")
+				.attr("text-anchor", (_, i) => i === 0 ? "start" : "end")
+				.attr("dx", (_, i) => i === 0 ? "-0.25em" : "0.25em")
+				.attr("visibility", "hidden");
+		});
+
+	// `hovered` signal gets updated in mousemove event
+	effect(() => {
+		// const fmt = d3.format("~s");
+		const fmt = tickFormatterForBins(type, bins);
+		hoveredTick.selectAll(".tick")
+			.attr("transform", `translate(${x(hovered.value || 0)},0)`)
+			.attr("visibility", hovered.value ? "visible" : "hidden");
+
+		hoveredTick
+			.selectAll(".tick text")
+			.text(`${fmt(hovered.value || 0)}`)
+			.attr("visibility", hovered.value ? "visible" : "hidden");
+
+		const hoveredTickText = hoveredTick.select(".tick text")
+			.node() as SVGGraphicsElement;
+		const bbox = hoveredTickText.getBBox();
+
+		svg
+			.selectAll(".hovered-bg")
```

Comment:
Don't know for sure but I think this could just be

```suggestion
		hoverBackground
```

if edit applied from above